.clang-format

```diff
@@ -31,13 +31,13 @@ BraceWrapping:
   AfterUnion:      false
   AfterExternBlock: false
   BeforeCatch:     false
-  BeforeElse:      false
+  BeforeElse:      true
```

Comment:
"} else {" is more compact, why changing it?

Reply:
I really don't like "} else {", because it looks confusing to me.